### ISSUE TITLE
Dark theme with white text

### ISSUE THREAD

**Issue by DrRob** [Kunzisoft/KeePassDX#620]
Can we have a dark theme with white text, please? The coloured text makes it look like a child's toy... :-)

**Comment by J-Jamet** [comment]
The standard text for dark themes is already white. Can you be more specific about the type of text you would like to change. You should make an image pattern or change the configuration to see what you want. (And I have the mind of a child, so I don't see where the problem is)

**Comment by DrRob** [comment]
Hi, I meant the "child's toy" reference to be lighthearted but on second reading that might be a bit insulting - apologies!  I should think twice before I type.

With the Night theme selected, for example, the names (and icons) of the list of top-level Groups are green.  It's only when I open a group and get to Entries that the text becomes white.

I admit it's a subjective thing, but then themes are subjective.  I would prefer that all the text and icons were black or white to contrast with the theme's light or dark background colour.  The ">" symbol is enough to indicate a Group.

I did look in the source code to see if I could change it myself, but I'm not familiar enough with Android app programming and the way resources work to know what to do, or have the build environment set up.  (It all looks impressively clean and tidy though...)

**Comment by J-Jamet** [comment]
No problem, it didn't shock me. :D 
True, the colors are relative, what you can do is a style file with a new theme that suits you. ;)

**Comment by DrRob** [comment]
OK, that's interesting.  Is that something that can be done without needing to rebuild the app?  Can you point me at an example?

**Comment by J-Jamet** [comment]
No, you must recompile the app, these files are part of the source code.
https://github.com/Kunzisoft/KeePassDX/blob/master/app/src/main/res/values/style_black.xml

**Comment by c-b-x** [comment]
Isn't this outdated since the introduction of Material You? My KeePassDX is completely monochrome in compliance with the settings of my phone.
